### PR TITLE
Browser Webcomponent Fixes

### PR DESCRIPTION
**Pull request by adamduren**
Include web components polyfill
Fix CSS encapsulation issue
Add Environment flag to conditionally load, Raven, Heap, and GA

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -29,6 +29,7 @@ keep_files:
   - elements/elements.html
   - bower_components/promise-polyfill/Promise.js
   - bower_components/raven-js/dist/raven.js
+  - bower_components/webcomponentsjs/webcomponents-lite.min.js
```
Comment (adamduren) [inline _config.yml]: This wasn't in this list so it was getting excluded by Jekyll

File: _includes/head.html [hunk 2]
```diff
@@ -31,20 +31,22 @@
...
+
   <script type="text/javascript">
     window.heap=window.heap||[],heap.load=function(e,t){window.heap.appid=e,window.heap.config=t=t||{};var r=t.forceSSL||"https:"===document.location.protocol,a=document.createElement("script");a.type="text/javascript",a.async=!0,a.src=(r?"https:":"http:")+"//cdn.heapanalytics.com/js/heap-"+e+".js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(a,n);for(var o=function(e){return function(){heap.push([e].concat(Array.prototype.slice.call(arguments,0)))}},p=["addEventProperties","addUserProperties","clearEventProperties","identify","removeEventProperty","setEventProperties","track","unsetEventProperty"],c=0;c<p.length;c++)heap[p[c]]=o(p[c])};
       heap.load("955190170");
   </script>
+  {% endif %}
 
   <script src="/bower_components/promise-polyfill/Promise.js"></script>
   <script src="/assets/scripts/index.js"></script>
-
```
Comment (adamduren) [inline _includes/head.html]: This wasn't used but it gave me a good idea on how to do the above.

File: _includes/head.html [hunk 3]
```diff
@@ -31,20 +31,22 @@
 
   <link rel="stylesheet" href="/bower_components/components-font-awesome/css/font-awesome.min.css">
 
+  {% if jekyll.environment == 'production' %}
```
Comment (adamduren) [inline _includes/head.html]: This way we won't get spammed while in development.

File: assets/styles/blog-index.scss [hunk 4]
```diff
@@ -12,6 +12,6 @@
   padding: 1rem;
 }
 
-.blog-tile {
```
Comment (adamduren) [inline assets/styles/blog-index.scss]: This was creeping in with Firefox which doesn't support shadow dom.

Comment (zacharytamas) [inline assets/styles/blog-index.scss]: I don't think we're actually using Shadow DOM though, Polymer defaults to Shady DOM.

File: _includes/head.html (hunk 2 above)
Comment (zacharytamas) [inline _includes/head.html]: Yeah, I had removed this before but we changed and merged so much I guess it didn't get persisted.

**Pull request merged**